src/main/java/wooteco/subway/line/LineService.java

```diff
@@ -1,65 +1,74 @@
 package wooteco.subway.line;
 
+import lombok.RequiredArgsConstructor;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
-import wooteco.subway.exception.line.LineNameDuplicatedException;
+import wooteco.subway.domain.Line;
+import wooteco.subway.domain.Section;
+import wooteco.subway.domain.Sections;
+import wooteco.subway.domain.Station;
+import wooteco.subway.exception.line.LineDuplicatedInformationException;
 import wooteco.subway.exception.line.LineNotFoundException;
-import wooteco.subway.line.dao.LineDao;
+import wooteco.subway.line.dao.JdbcLineDao;
+import wooteco.subway.line.web.LineRequest;
+import wooteco.subway.section.SectionService;
+import wooteco.subway.station.StationService;
 
 import java.util.List;
 
-@Transactional(readOnly = true)
+@RequiredArgsConstructor
 @Service
 public class LineService {
+    private final JdbcLineDao lineDao;
```

Comment:
```suggestion
    private final LineDao lineDao;
```

추상화한 객체로 인스턴스 변수를 두고 JdbcLineDao를 생성자를 사용해서 의존성 주입 받으세요
지금은 jdbc를 쓰고 있지만 운영하다보면 다른 라이브러리를 쓸 수도 있을텐데 지금처럼 직접 JdbcLineDao를 지정하면 수정할 부분이 많아집니다.

Reply:
인터페이스로 다시 추출해주었어요!ㅎㅎㅎ